tokio/src/net/unix/pipe.rs

```diff
@@ -16,6 +16,33 @@ cfg_io_util! {
     use bytes::BufMut;
 }
 
+/// Creates a new anonymous Unix pipe.
+///
+/// This function will open a new pipe and associate both pipe ends with the default
+/// event loop.
+///
+/// If you need to create a pipe for communicating with a spawned process, you can
+/// also see how to use `Stdio::piped()` with [`tokio::process`].
+///
+/// [`tokio::process`]: crate::process
```

Comment:
I don't understand this sentence. Are you saying that if they want to do this, they should read the source code of `tokio::process` to figure out how?

Reply:
I would like to encourage users to first see if using a combination like `Command::stdout(Stdio::piped())` and `ChildStdout` is enough for their use case of anonymous pipes when spawning a process. This interface has been in tokio and the standard library for ages, and is arguably easier to use than `tokio::net::unix::pipe`, although it's obviously limited to stdin/stdout/stderr. However this module will be the first to show up if a user tries to search "pipe" in tokio's docs.

In the module level documentation of [`tokio::process`](https://docs.rs/tokio/1.33.0/tokio/process/index.html#examples) there are examples using `Stdio::piped()`, but I don't know how to directly link to them.

Reply:
As a side note, we should document the fact that `ChildStdout::into_owned_fd` and others set the blocking mode.